packages/client/src/apis/market/MarketQueryAPI.ts

```diff
@@ -50,15 +50,15 @@ export class MarketQueryAPI extends BaseVertexAPI {
   /**
    * @description Queries engine to determine if the order can be submitted within health requirements
    */
-  async validateOrderParams(params: ValidateEngineOrderParams) {
+  async validateOrderParams(params: EngineQueryValidateOrderParams) {
```

Comment:
would it be possible to drop the `Engine` prefix in all these? seems it would be more standard with the rest

Reply:
Then we need to redeclare something like `type ValidateOrderParams = EngineQueryValidateOrderParams`